src/Uno.UI.Toolkit/ElevatedView.cs

```diff
@@ -212,6 +214,11 @@ private void UpdateElevation()
 
 		protected override Windows.Foundation.Size ArrangeOverride(Windows.Foundation.Size finalSize)
 		{
+#if __ANDROID__
+			_invalidateShadow = true;
+			((ViewGroup)this).Invalidate();
```

Comment:
... and you're not causing a layout loop?

Reply:
This occurs only on ElevatedView property changes, and based on testing does not seem to cause issues... so far 😂